### DISCUSSION TITLE
My build hanging in queued status forever

### DISCUSSION THREAD

**Discussion by jingp-ofload** [codemagic-ci-cd/codemagic-docs#1824] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

It doesn't matter if I triggered the build from `new build` or `branch push`. The build just hangs at `queued` status.

### Expected results

The build should be executed.

### Actual results

Always `queued`

### Build id (optional)

649bad29b82b2dacaf4830b5

**Comment by ferdiangunawan** [comment]
same here. seems like all the users are running the same issue

**Comment by jingp-ofload** [comment]
@helinanever Not sure if I'am cuing right person. Is it possible to publish the service status somewhere?

**Comment by emappstore777** [comment]
Seems like there some problems with Apple server. 5-6 hours ago, when building worked - in publishing received error 500 (server unavailable)

**Reply by jingp-ofload** [reply]
@emappstore777  Thanks for your reply. Unfortunately my andorid build is stuck as well.

**Reply by emappstore777** [reply]
I think it is a general issue, same: iOS build stuck in queue

**Reply by jingp-ofload** [reply]
At least I got something to explain to my manager.

**Comment by shrihari2i** [comment]
@jpAtOfload I have also same issue from today morning we are also use React Native. Hope its solve asap

**Reply by maks** [reply]
Hi, thank you for reporting the issue. There does seem to be an issue at the moment with builds not running, we're looking into it now.

**Reply by jingp-ofload** [reply]
@maks I used to be with CircleCI but I really like the way Codemagic manages the iOS certificates and Android keys.
Please, if possible, publish the service status somewhere or put a rough availabilities figure somewhere.

**Reply by shrihari2i** [reply]
yes i totally agree with @jpAtOfload you need to update on tweeter or your website so user get idea @maks

**Reply by shrihari2i** [reply]
@maks Hey my job now running look like issue is solve

**Comment by joseignacioojeda** [comment]
@maks, Could you give us an estimate of when this issue will be solved?

All our builds are blocked and we are in the release process.

Thanks in advance

**Comment by helinanever** [comment]
There are issues with build machine availability.
* Apple silicon (M1/M2) machines are available but there may be longer build queues.
* Linux and Windows machines are currently unavailable. We are working on restoring them.

We will post another update shortly and apologize for any inconvenience caused.

**Comment by aarajput** [comment]
Every week i hear that codemagic machines are down. I am not sure now if code magic is reliable or not. 
It been more than 4 hours but still no update.

**Comment by teddywangsg** [comment]
My builds are recovered already 😁

**Comment by helinanever** [comment, marked as answer]
Hi everyone, please see the incident report here: https://github.com/orgs/codemagic-ci-cd/discussions/1827
